library/src/window_impl.cxx

```diff
@@ -122,6 +115,20 @@ window& window_impl::setSize(int width, int height)
   return *this;
 }
 
+//----------------------------------------------------------------------------
+window& window_impl::setPosition(int x, int y)
+{
+#if __APPLE__
```

Comment:
It's not related to Apple, but Cocoa. What about testing `RenWin->IsA("vtkCocoaRenderWindow")`?

Reply:
bad for coverage, but ok :p